### DISCUSSION TITLE
Change the title of the rendered version to indicate that the project is still active and there are ongoing updates

### DISCUSSION THREAD

**Discussion by annamariaschleimer** [International-Data-Spaces-Association/ids-specification#150] [General]
Hello everyone,

When sharing the link to the documentation https://docs.internationaldataspaces.org/ids-knowledgebase/v/dataspace-protocol/overview/readme, the first header says "Working Draft 1 February 2023", even though there are frequent updates in the repository. This sometimes leads to confusion about whether the DSP is still being worked on (since it does not seem to have been updated since February), and it is not always clear to (uninvolved, external) readers that the content is changing according to the latest updates in the main branch.

Suggestion: Change the heading to something like "V.08 with ongoing changes" or similar. 
There could also be a changelog or some indicator of what has happened since February, which would not make the version number questionable, but still indicate progress. 

Best,
Anna

**Comment by jimmarino** [comment]
We could put a link to the GitHub repository. However, I don't think we should update the linked documentation without going through a formal release process and incrementing the version. In other words, all published versions should be static.

**Reply by juliapampus** [reply]
> In other words, all published versions should be static.

The GitBooks (rendered version discoverable under the above mentioned link) are updated on each merge to `main`. This is a problem then, right?

**Reply by juliapampus** [reply]
@AScFH done.
